pom.xml

```diff
@@ -32,6 +32,17 @@
             <artifactId>quartz</artifactId>
             <version>2.1.5</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>2.6.1</version>
```

Comment:
Please use the newest version of log4j, there is already version 2.11.x in maven repo. 2.6.1 is from 2016 year.